MobileWallet/TariLib/Wrappers/Utils/MicroTari.swift

```diff
@@ -43,62 +43,91 @@ import Foundation
+    private static let preciseWithOperatorFormatter = NumberFormatter()
+    private static let editFormatter = NumberFormatter()
 
-    let rawValue: UInt64
+    public static var groupingSeparator: String {
+        return defaultFormatter.groupingSeparator
+    }
+
+    public static var decimalSeparator: String {
+        return defaultFormatter.decimalSeparator
+    }
+
+    var rawValue: UInt64
 
     var taris: Float {
         return Float(self.rawValue) / Float(MicroTari.CONVERSION)
     }
 
     var formatted: String {
-        return defaultFormatter.string(from: NSNumber(value: self.taris))!
+        return MicroTari.defaultFormatter.string(from: NSNumber(value: self.taris))!
     }
 
     var formattedWithOperator: String {
-        return withOperatorFormatter.string(from: NSNumber(value: self.taris))!
+        return MicroTari.withOperatorFormatter.string(from: NSNumber(value: self.taris))!
     }
 
     var formattedWithNegativeOperator: String {
-        return withOperatorFormatter.string(from: NSNumber(value: self.taris * -1))!
+        return MicroTari.withOperatorFormatter.string(from: NSNumber(value: self.taris * -1))!
     }
 
     var formattedPrecise: String {
-        return preciseFormatter.string(from: NSNumber(value: self.taris))!
+        return MicroTari.preciseFormatter.string(from: NSNumber(value: self.taris))!
     }
 
     var formattedPreciseWithOperator: String {
-        return preciseWithOperatorFormatter.string(from: NSNumber(value: self.taris))!
+        return MicroTari.preciseWithOperatorFormatter.string(from: NSNumber(value: self.taris))!
     }
 
     init(_ rawValue: UInt64) {
         self.rawValue = rawValue
 
-        defaultFormatter.numberStyle = .decimal
-        defaultFormatter.minimumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        defaultFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        defaultFormatter.negativePrefix = "-"
-
-        withOperatorFormatter.numberStyle = .decimal
-        withOperatorFormatter.minimumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        withOperatorFormatter.maximumFractionDigits = MicroTari.ROUNDED_FRACTION_DIGITS
-        withOperatorFormatter.positivePrefix = "+ "
-        withOperatorFormatter.negativePrefix = "- "
-
-        preciseFormatter.numberStyle = .decimal
-        preciseFormatter.minimumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseFormatter.maximumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseFormatter.negativePrefix = "- "
-
-        preciseWithOperatorFormatter.numberStyle = .decimal
-        preciseWithOperatorFormatter.minimumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseWithOperatorFormatter.maximumFractionDigits = MicroTari.PRECISE_FRACTION_DIGITS
-        preciseWithOperatorFormatter.positivePrefix = "+ "
-        preciseWithOperatorFormatter.negativePrefix = "- "
+        MicroTari.defaultFormatter.numberStyle = .decimal
```

Comment:
If these are static vars, do they need to be set on each init? Is there a way to just set once?